salvo/src/lib/common/test_file_ops.py

```diff
@@ -0,0 +1,89 @@
+
+"""
+Test source_tree operations needed for executing benchmarks
+"""
+import pytest
+
+import os
+import json
+import tempfile
+
+from src.lib.common import file_ops
+
+from unittest import mock
+
+
+_test_json = """
```

Comment:
(nit) Constants should use CAPS_WITH_UNDER in their names, so this and the following constant should be named:

_TEST_JSON
_TEST_YAML

https://github.com/google/styleguide/blob/gh-pages/pyguide.md#3164-guidelines-derived-from-guidos-recommendations

Reply:
I'll take care of it here.